### DISCUSSION TITLE
When we tap a notification we should be taken to the bottom of the issue not the top

### DISCUSSION THREAD

**Discussion by jidanni** [community/community#8190] [Mobile]
Because the bottom of the issue is where the action is at, that we are being notified about. And it might take miles of scrolling to get there otherwise. Actually it should take us right to the point of what we are being notified about.

**Comment by eliperkins** [comment]
Thanks for the feedback @jidanni ! This behavior should currently exist for _unread_ notifications (where the app scrolls to the latest unread content, usually at the bottom of the issue). I understand that this only works once, and that after reading, you'll need to scroll a bit to get to the bottom.

Can I get some more details from you on what platform (iOS or Android) that you're experiencing this on? What version of the app as well?

**Reply by jidanni** [reply]
Android.
Latest beta app.

**Comment by jidanni** [comment]
BTW, the user might click the same link several times, or even share it,
so would expect it to take him and others to the same place each time.
